basis/compiler/cfg/gc-checks/gc-checks.factor

```diff
@@ -122,9 +118,7 @@ M: ##box-displaced-alien allocation-size* drop 5 cells ;
 
 PRIVATE>
 
-:: insert-gc-checks ( cfg -- )
-    cfg blocks-with-gc [
-        cfg needs-predecessors
-        [ process-block ] each
-        cfg cfg-changed
-    ] unless-empty ;
+: insert-gc-checks ( cfg -- )
+    [ needs-predecessors ]
```

Comment:
This behavior is a little different, does it matter that it previously only applied if ``blocks-with-gc`` was not empty, and now applies all the time?